### DISCUSSION TITLE
Development progress

### DISCUSSION THREAD

**Discussion by insel-maz** [PinJuke/PinJuke#1] [Announcements]
Work on the file browser overlay is well advanced. See original screenshot:
![grafik](https://github.com/PinJuke/PinJuke/assets/1782877/e6216003-49d2-4c90-830b-25a28455d687)

**Comment by insel-maz** [comment]
Visualizations are now playing and DOF can be controlled:
![DOF](https://github.com/PinJuke/PinJuke/assets/1782877/d95b419d-4b2c-4257-a166-c601b9add1b8)

**Comment by VirtualPinballSpreadsheet** [comment]
Just wanted to say thank you for developing PinJuke and keep up the great job. I hope more updates will follow.

**Comment by insel-maz** [comment]
@VirtualPinballSpreadsheet Thank you for your kind words.

At the moment I am reworking the visualizations (integration of projectM). See original screenshot:
![Reworked projectM](https://github.com/user-attachments/assets/3f7f66e9-0ddf-4458-80af-fe09ed0582c0)

**Reply by VirtualPinballSpreadsheet** [reply]
Nice! Those certainly compliment the videos and songs with those great visuals.

I promoted your program on Virtual Pinball Chat on Discord a while ago and also added it to a 'Virtual Pinball Software' spreadsheet which covers must have VP and related software. Hope you don't mind.

Keep up the great work! People are very excited for your video and mp3 jukebox.
